packages/core/src/Synchronizer.ts

```diff
@@ -321,6 +357,7 @@ export class Synchronizer extends EventEmitter {
      * block we'll poll many times quickly
      */
     async start() {
+        const N = 20
```

Comment:
it can be public variable like 
https://github.com/Unirep/Unirep/blob/19e489ccaed0392765654ae743f2c2d04f257358/packages/core/src/Synchronizer.ts#L47
so it can be changed in different synchronizer?

Reply:
switched `const N = 20` to use `blockRate` since it wasn't being used elsewhere